Rakefile

```diff
@@ -28,7 +28,14 @@ namespace :lint do
     RakeUtils.bundle_exec 'haml-lint dashboard pegasus'
   end
 
-  task all: [:ruby, :haml]
+  task :javascript do
+    Dir.chdir(apps_dir) do
+      # lint all js/jsx files in dashboardd/app/assets/javascript
+      RakeUtils.system 'grunt jshint:files --glob "../dashboard/app/assets/javascripts/**/*.js*(x)"'
```

Comment:
Unclear. On the one hand, that seems like a pretty reasonable expectation. On the other, we do call rake lint as part of our websites build it looks like, and this would be redoing work we've already done (as part of apps and shared builds). This might be a small enough thing that we don't care about the duplication.